runtime/onert/core/include/compiler/StaticShapeInference.h

```diff
@@ -55,16 +55,30 @@ class StaticInferer : public ir::OperationVisitor
    *        when running kernel.
    * @param op_seq sequence of operations
    */
-  void infer(const ir::OpSequence &op_seq)
+  bool infer(const ir::OpSequence &op_seq)
   {
+    bool ret_has_dynamic_tensor = false;
+
+    // TODO Eliminate _has_dynamic_tensor by, e.g., introducing bool accept(..)
```

Comment:
I personally prefer to not introducing `bool accept()`. We need other return types in other part of code. But it is not worth introducing all return types we need. So I ended up with having only one `accept` function.